backend/go.mod

```diff
@@ -0,0 +1,22 @@
+module github.com/asgardeo/thunder/backend
```

Comment:
```suggestion
module github.com/asgardeo/thunder/backend
```

What if we go with `github.com/asgardeo/thunder` for the module name?

Reply:
Fixed with: https://github.com/asgardeo/thunder/compare/3803d62d852511eea34950e4b51bfcaec46b21ea..e90aa8acfa5d58be27804d0cf7d5b2c136008a4d